docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -1508,7 +1508,7 @@ However, in some situations, these classes cause the `JAXBContext` to fail: for
 
 [NOTE]
 ====
-The property `quarkus.jaxb.exclude-classes` accepts a comma separated list of fully qualified class names, for example: `quarkus.jaxb.exclude-classes=org.acme.one.Model,org.acme.two.Model`. 
+The property `quarkus.jaxb.exclude-classes` accepts a comma separated list of fully qualified class names or a package name ending with `.*` as wildcard which will respect all classes in that package and subpackages, for example: `quarkus.jaxb.exclude-classes=org.acme.one.Model,org.acme.two.Model,org.acme.somemodel.*`. 
```

Comment:
I would go with the following, does it work for you?
```suggestion
The property `quarkus.jaxb.exclude-classes` accepts a comma separated list of either fully qualified class names
or package names. Package names must be suffixed by `.*` and all classes in the specified package and its subpackages will be excluded.

For instance, when setting `quarkus.jaxb.exclude-classes=org.acme.one.Model,org.acme.two.Model,org.acme.somemodel.*`, the following elements are excluded:

- The class `org.acme.one.Model`
- The class `org.acme.two.Model`
- All the classes in the `org.acme.somemodel` package and its subpackages
```

Reply:
+100

Reply:
Okay, thats detailed.

Reply:
Yeah, I wanted to be extra clear as the `.*` thing is not exactly obvious. It doesn't really work as a wildcard typically (or it would work for classes too), which is what you would expect.